code/controllers/subsystem/events.dm

```diff
@@ -204,6 +216,19 @@ SUBSYSTEM_DEF(events)
 		html += "</table>"
 		html += "</div>"
 
+		html += "<div class='block'>"
+		html += "<h2>Event Modifiers</h2>"
+		html += "Currently applied event modifiers."
+		html += "<table[table_options]>"
+		html += "<tr[head_options]><td[row_options1]>Modifier</td><td[row_options3]></td><td><A align='right' href='?src=[UID()];add_modifier=1'>Add</A></td></tr>"
+		for(var/SP in shared_special_event_modifiers)
+			html += "<tr>"
+			html += "<td>[SP]</td>"
+			html += "<td><A align='right' href='?src=[UID()];remove_modifier=[SP]'>Remove</A></td>"
+			html += "</tr>"
+		html += "</table>"
+		html += "</div>"
```

Comment:
```suggestion
		var/uid_cache = UID()
		html += "<tr[head_options]><td[row_options1]>Modifier</td><td[row_options3]></td><td><A align='right' href='?src=[uid_cache];add_modifier=1'>Add</A></td></tr>"
		for(var/SP in shared_special_event_modifiers)
			html += "<tr>"
			html += "<td>[SP]</td>"
			html += "<td><A align='right' href='?src=[uid_cache];remove_modifier=[SP]'>Remove</A></td>"
			html += "</tr>"
		html += "</table>"
		html += "</div>"
```
Caching this can prove good if this loop ever gets huge